### PR TITLE
Fix #1: Spaces in Hollywood executable are not escaped

### LINKED ISSUE #1: Spaces in path not escaped
Great work implementing this Extension!

I noticed that on Windows, the current example file for a Build Task, doesn't quite work.
The spaces in the path to the Hollywood executable are not escaped, so the task tries to execute `C:\Program` since that's where the first space shows up.

It might be better to break the `command` and the `args` in separate fields, instead:

```
"command": "${config:hollywood.exePath}",
            "args": ["${workspaceFolder}/${config:hollywood.mainFile}"],
```

The above works normally with the spaces in the path. It also looks a bit nicer, since you don't have one very long line in a single command. ;-)

### PR DESCRIPTION
**Pull request by midwan**
Here's a suggested fix, to break the command and arguments in separate fields.
This way, VSCode takes care of any spaces necessary and the command works properly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,7 +158,8 @@ This is a complete example of a task definition:
 {
     "label": "Run Hollywood Main script",
     "type": "shell",
-    "command": "${config:hollywood.exePath} ${workspaceFolder}/${config:hollywood.mainFile}",
+    "command": "${config:hollywood.exePath}",
+    "args":["${workspaceFolder}/${config:hollywood.mainFile}"],
```
Comment (JohnArcher) [inline README.md:162]: As you mentioned it in #1 could you update your PR and delete the ${workspaceFolder}/ string? Thank you. I think this is easier to merge then (I am not super experienced with GitHub PRs but I think this is the correct workflow. Correct me, if I am wrong).

File: exampleFiles/tasks.json [hunk 2]
```diff
@@ -6,7 +6,8 @@
         {
             "label": "Run Hollywood Main script",
             "type": "shell",
-            "command": "${config:hollywood.exePath} ${workspaceFolder}/${config:hollywood.mainFile}",
+            "command": "${config:hollywood.exePath}",
+            "args": ["${workspaceFolder}/${config:hollywood.mainFile}"],
```
Comment (JohnArcher) [inline exampleFiles/tasks.json:10]: As you mentioned it in #1 could you update your PR and delete the ${workspaceFolder}/ here and on the other locations here in this file? Thank you!

**Comment by midwan** [comment]
@JohnArcher 
I've updated my changes, by removing the workspace folder :)

File: exampleFiles/tasks.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
             "label": "Run Hollywood Main script",
             "type": "shell",
             "command": "${config:hollywood.exePath}",
-            "args": ["${workspaceFolder}/${config:hollywood.mainFile}"],
+            "args": ["$${config:hollywood.mainFile}"],
```
Comment (JohnArcher) [inline exampleFiles/tasks.json:10]: Ooops, there are two `$` signs! :-) -> `$${config:hollywood.mainFile}`

Comment (midwan) [reply exampleFiles/tasks.json]: Sorry about that, fixed it with a new commit :)

Comment (JohnArcher) [reply exampleFiles/tasks.json]: No problem. :-) Thanks a lot for your work!

**Pull request merged**